api-inbound.yaml

```diff
@@ -98,7 +98,17 @@ paths:
           async: false
           request: '#! json .Request.Body !#'
           success: '{"status": 204, "bodyPassthrough": true}'
-          error: '{"status": 500, "bodyPassthrough": true}'
+          error: >
+            {
+              "status":
+                #! if eq .Response.Body.errorType "ErrorFetchingAssets" !# 502,
+                #! end !#
+              "body": {
+                #! if eq .Response.Body.errorType "ErrorFetchingAssets" !#
+                "code": 502, "status": "Bad Gateway",
```

Comment:
yea it looks like it's undefined. Should it be defined? Because i believe the handler only can return nil, or this type of error, so there it's impossible to have any other type of error returned

Reply:
Task: hmm, I suspect that a different error would be returned in the (admittedly rare, but it could happen) case of a panic or some other completely unexpected failure. Please add an else clause that returns a [500 Internal Server Error](https://http.cat/500)